src/workloads/execution/TimeSeriesGroupDateTrunc.yml

```diff
@@ -0,0 +1,113 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  Timeseries performance test
+
+Database: &db timeseries
+Collection: &coll Collection0
+
+TimeField: &time time
+MetaField: &meta meta
+ValueField: &value value
+DollarTimeField: &dollar_t {^Join: {array: [$, *time]}}
+DollarMetaField: &dollar_m {^Join: {array: [$, *meta]}}
+DollarValueField: &dollar_value {^Join: {array: [$, *value]}}
+
+MetaCount: &meta_count 10
+DocumentCount: &document_count 1000000
+TestRepeat: &test_repeat 10
+
+Actors:
+- Name: Setup
+  Type: RunCommand
+  Thread: 1
+  Phases:
+  - Repeat: 1
+    Database: *db
+    Operations:
+      - OperationName: RunCommand
```

Comment:
Can you add a comment that the drop command is only here as a convenience to support running the test locally.  A sys-perf run would have an empty database.